bin/source_of_truth/commands_source_of_truth.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2023 Sharezone UG (haftungsbeschränkt)
+# Licensed under the EUPL-1.2-or-later.
+#
+# You may obtain a copy of the Licence at:
+# https://joinup.ec.europa.eu/software/page/eupl
+#
+# SPDX-License-Identifier: EUPL-1.2
+
+# This file is the "source of truth" for running commands that are the same for all operating systems.
+# We want to run the same command everywhere, on local dev machines and on CI systems.
+# Since Sharezone can be developed on Linux / Mac / Windows just putting the command in a a .sh file isn't enough, since Windows couldn't execute it.
+# 
+# CI:
+# When running these commands via CI we just read them in inside the bash script and execute the command.
+# This saves us first installing Dart for e.g. just executing a simple go script.
```

Comment:
Can we keep a line of length of 80 for the comments? I'm using the "Rewrap" extension to format comments